stable/agent/values.yaml

```diff
@@ -154,6 +154,14 @@ rbac:
           - create
           - update
 
+horizontalPodAutoscaler:
+  enabled: false
+  # A list of types to use for autoscaling.
+  # Can be cpu, memory, or custom.
```

Comment:
See my other comment, but having it limited like this really reduces the usability of this entire feature

Reply:
I've resolved this in the latest commit. See below.